### PR TITLE
ubports-installer: Add version 0.7.4-beta

### PR DESCRIPTION
**Pull request by schance995**
App looks like it works, and doesn't look like there is persistent data.
The app works fine without extraction but runs faster with it. If this is not ok then I can change it.

### CODE REVIEW

File: bucket/ubports-installer.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    ],
+    "checkver": {
+        "url": "https://github.com/ubports/ubports-installer/releases/latest",
+        "regex": "ubports\\-installer_(?<version>[\\d.]+)\\-beta_win\\.exe"
+    },
+    "autoupdate": {
+        "url": "https://github.com/ubports/ubports-installer/releases/download/$version-beta/ubports-installer_$version-beta_win.exe#/dl.7z",
+        "hash": {
+            "url": "$baseurl/ubports-installer_$version-beta_win.exe.sha256"
+        }
```
Comment (Ash258) [inline bucket/ubports-installer.json:28]:
```suggestion
    "version": "0.7.4-beta",
    "description": "A friendly cross-platform Installer for Ubuntu Touch",
    "homepage": "https://ubuntu-touch.io//ubuntu-touch.io",
    "license": "GPL-3.0-only",
    "suggest": {
        "adb": "adb"
    },
    "architecture": {
        "64bit": {
            "url": "https://github.com/ubports/ubports-installer/releases/download/0.7.4-beta/ubports-installer_0.7.4-beta_win.exe#/dl.7z",
            "hash": "ec347f98c02ec0b9a339e5c1d35b35da6de8f79509a1f8c065ecff037008e747",
            "pre_install": [
                "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
                "Remove-Item \"$dir\\`$*\" -Recurse"
            ]
        }
    },
    "shortcuts": [
        [
            "ubports-installer.exe",
            "UBports Installer"
        ]
    ],
    "checkver": {
        "url": "https://github.com/ubports/ubports-installer/releases/latest",
        "regex": "ubports-installer_([\\w.-]+)_win\\.exe"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/ubports/ubports-installer/releases/download/$version/ubports-installer_$version_win.exe#/dl.7z",
                "hash": {
                    "url": "$baseurl/ubports-installer_$version_win.exe.sha256"
                }
            }
        }
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Review by Ash258 — COMMENTED** [review]
Manifest and installation is fine, but I cannot fully test the application

**Comment by NeoTheThird** [comment]
> doesn't look like there is persistent data

Upstream maintainer here. There is persistent data in `%APPDATA%`, but if it doesn't persist, that's okay. You'll just have to re-download files on every start.

**Comment by issaclin32** [comment]
> > doesn't look like there is persistent data
> 
> Upstream maintainer here. There is persistent data in `%APPDATA%`, but if it doesn't persist, that's okay. You'll just have to re-download files on every start.

Thanks for the info. I will use **Junctions** to keep the data (when user updates the app with Scoop)

**Comment by issaclin32** [comment]
Superseded by #7630.

**Pull request closed without merging**